baxter_collaboration_lib/test/test_ros_thread_image.cpp

```diff
@@ -118,6 +136,32 @@ TEST(rosimagetest, testinternalthreadentry)
     EXPECT_EQ(rtii.centroid(), cv::Point(100, 100));
 }
 
+TEST(rosimagetest, testmono8image)
+{
+    // Creates an object that sends a 200x200 black image with a red circle overlaid
+    ROSThreadImageTester rtit("test");
+
+    // Creates an object responsible for receiving an image
+    // and finding the centroid of a contour
+    ROSThreadImageInstance rtii("test");
```

Comment:
Here you're creating the same instnace as before, ie an `rtii` with bgr8 encoding. So your're not testing against the `mono8` version.